### PR TITLE
en api docs for primitives

### PR DESCRIPTION
**Pull request by JayceLai**

### CODE REVIEW

**Comment by JayceLai** [comment]
https://github.com/cocos-creator/3d-tasks/issues/2506

File: cocos/core/primitive/box.ts [hunk 1]
```diff
@@ -62,7 +64,7 @@ interface IBoxOptions extends RecursivePartial<IGeometryOptions> {
 /**
  * @en
  * This function generates a box with specified extents and centered at origin,
- * but may be repositioned through `center` option).
+ * but may be repositioned through `center` option.
```
Comment (slackmoehrle) [inline cocos/core/primitive/box.ts]: through the `center`

File: cocos/core/primitive/capsule.ts [hunk 2]
```diff
@@ -19,13 +21,15 @@ const temp1 = new Vec3(0, 0, 0);
 const temp2 = new Vec3(0, 0, 0);
 
 /**
+ * Generate a capsule with radiusTop radiusBottom 0.5, height 2, centered at origin,
+ * but may be repositioned through `center` option).
```
Comment (slackmoehrle) [inline cocos/core/primitive/capsule.ts]: through the `center`

File: cocos/core/primitive/circle.ts [hunk 3]
```diff
@@ -27,7 +29,8 @@ function applyDefaultCircleOptions (options?: RecursivePartial<ICircleOptions>):
 
 /**
  * @en
- * Generate a circle with radius 1, centered at origin.
+ * Generate a circle with radius 1, centered at origin,
+ * but may be repositioned through `center` option.
```
Comment (slackmoehrle) [inline cocos/core/primitive/circle.ts]: through the `center`

File: cocos/core/primitive/cone.ts [hunk 4]
```diff
@@ -8,13 +8,16 @@ import { IGeometry } from './define';
 type IConeOptions = ICylinderOptions;
 
 /**
+ * @en
+ * Generate a cone with radius 0.5, height 1, centered at origin,
+ * but may be repositioned through `center` option.
```
Comment (slackmoehrle) [inline cocos/core/primitive/cone.ts]: through the `center`

File: cocos/core/primitive/cylinder.ts [hunk 5]
```diff
@@ -6,27 +6,32 @@ import { Vec3 } from '../math';
...
+  arc: number;
 }
 
 const temp1 = new Vec3(0, 0, 0);
 const temp2 = new Vec3(0, 0, 0);
 
 /**
+ * @en
+ * Generate a cylinder with radiusTop radiusBottom 0.5, height 2 and centered at origin,
+ * but may be repositioned through `center` option.
```
Comment (slackmoehrle) [inline cocos/core/primitive/cylinder.ts]: through the `center`

**Review by slackmoehrle — CHANGES_REQUESTED** [review]

**Review by slackmoehrle — APPROVED** [review]

**Pull request merged**